Gopkg.toml

```diff
@@ -2,10 +2,26 @@
   name = "golang.org/x/text"
   version = "0.3.0"
 
+
+[[override]]
+  name = "k8s.io/client-go"
+  # revision for tag "kubernetes-1.13.1"
+  revision = "8d9ed539ba3134352c586810e749e58df4e94e4f"
+
+[[override]]
+  name = "github.com/coreos/prometheus-operator"
+  version = "=v0.26.0"
+
+[[override]]
+  name = "sigs.k8s.io/controller-runtime"
+  version = "=v0.1.10"
+
+
+
 [[constraint]]
```

Comment:
This is a duplicate?

Reply:
Is there a reason for any override in here? Should probably just use constraint.